reflection/test/serverreflection_test.go

```diff
@@ -605,9 +605,9 @@ func testFileContainingSymbol(t *testing.T, stream v1reflectiongrpc.ServerReflec
 
 func testFileContainingSymbolError(t *testing.T, stream v1reflectiongrpc.ServerReflection_ServerReflectionInfoClient) {
 	for _, test := range []string{
-		"grpc.testing.SearchService",
+		"grpc.testing.SerchService",
```

Comment:
Sorry, but in the interest of not repeating this mistake, could you make these not "typos", but valid words that are just not correct?

E.g. "grpc.testing.OtherService" and "grpc.other.SearchResponse"?

Thanks!

Reply:
Sure, are you ok with using a couple of superfluous underscores? Force pushed and tests are passing locally.

Reply:
Yes, that's fine too, thanks!